packages/nuka/src/utils.ts

```diff
@@ -33,10 +33,6 @@ export const addEvent = (
   }
   if (elem.addEventListener) {
     elem.addEventListener(type, eventHandler, false);
-  } else if (elem.attachEvent) {
-    elem.attachEvent(`on${type}`, eventHandler);
-  } else {
-    elem[`on${type}`] = eventHandler;
```

Comment:
These are Internet Explorer methods that don't exist in modern APIs. We could polyfill the types but I think it's better to drop.